profiles/__init__.py

```diff
@@ -0,0 +1 @@
+from .factory import create_app
```

Comment:
this relative import makes you able to run `from profile import create_app` rather than `from profiles.factory import create_app`. So it's an additional layer of indirection, but not strictly needed